internal/pkg/storageos/error_format.go

```diff
@@ -0,0 +1,21 @@
+package storageos
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ListErrors is a basic formatter that outputs the number of errors
+// that occurred along with a list of the errors.
+func ListErrors(es []error) string {
+	if len(es) == 1 {
```

Comment:
Good point, previously the code would output with bullet points if there was more than one.  It can be simplified now it doesn't.

Reply:
Actually, probably best to leave as-is, otherwise we can't pluralize "error/errors" properly:

`1 error occurred`
`N errors occurred`

That said, I'm not sure we need to output the `1 error occurred: ` if there was only one...

Reply:
I've removed the prefix when there's only one error.  Then behaves as it would if multierror wasn't used.